src/Illuminate/Support/Number.php

```diff
@@ -150,6 +150,17 @@ public static function fileSize(int|float $bytes, int $precision = 0, ?int $maxP
         return sprintf('%s %s', static::format($bytes, $precision, $maxPrecision), $units[$i]);
     }
 
+    /**
+     * Divide a number.
+     *
+     * @param  int|float  $numerator
+     * @param  int  $denominator
+     */
+    public static function divide(int|float $numerator, int $denominator)
```

Comment:
The denominator could also be a `float`, no? :octocat:

Reply:
@JesusValera Good point! Thanks for the suggestion, initially didn't even think of that, I've made that change and pushed!